test/acceptance/features/bindAppToService.feature

```diff
@@ -723,3 +723,47 @@ Feature: Bind an application to a service
         Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-request-cross-ns-service" should be changed to "True"
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-cross-ns-service" should be changed to "True"
         And The application env var "BACKEND_HOST_CROSS_NS_SERVICE" has value "cross.ns.service.stable.example.com"
+
+    Scenario: Bind an application to a openshift route
+        * The openshift route is present
+            """
+            kind: Route
+            apiVersion: route.openshift.io/v1
+            metadata:
+                name: my-route
+                annotations:
+                    openshift.io/host.generated: 'true'
+                    service.binding/host: path={.spec.host} #annotate here.
+            spec:
+                host: example-sbo.apps.ci-ln-smyggvb-d5d6b.origin-ci-int-aws.dev.rhcloud.com
```

Comment:
Would this kind of host work every cluster? On OpenShift CI? On CRC? On any other ordinary OpenShift cluster?

Reply:
I guess yes. When we expose the service in openshift it has a similar format.